bundles/org.openhab.binding.miio/src/main/resources/database/dreame.vacuum.p2029-miot.json

```diff
@@ -0,0 +1,1076 @@
+				"channel": "brush_left_time1",
+				"type": "Number:Time",
+				"unit": "hours",
+				"stateDescription": {
+					"minimum": 0,
+					"maximum": 200,
+					"step": 1,
+					"pattern": "%.0f %unit%",
+					"readOnly": true
+				},
+				"refresh": true,
+				"actions": []
+			},
+			{
+				"property": "brush-life-level1",
+				"siid": 10,
+				"piid": 2,
+				"friendlyName": "Side Cleaning Brush - Brush Life Level",
+				"channel": "brush_life_level1",
+				"type": "Number:Dimensionless",
+				"unit": "percentage",
+				"stateDescription": {
+					"minimum": 0,
+					"maximum": 100,
+					"step": 1,
+					"pattern": "%.0f %unit%",
+					"readOnly": true
+				},
+				"refresh": true,
+				"actions": []
+			},
+			{
+				"property": "filter-life-level",
+				"siid": 11,
+				"piid": 1,
+				"friendlyName": "Filter - Filter Life Level",
+				"channel": "filter_life_level",
+				"type": "Number:Dimensionless",
+				"unit": "percentage",
+				"stateDescription": {
+					"minimum": 0,
+					"maximum": 100,
+					"step": 1,
+					"pattern": "%.0f %unit%",
+					"readOnly": true
+				},
+				"refresh": true,
+				"actions": []
+			},
+			{
+				"property": "filter-left-time",
+				"siid": 11,
+				"piid": 2,
+				"friendlyName": "Filter - Filter Left Time",
+				"channel": "filter_left_time",
+				"type": "Number:Time",
+				"unit": "hours",
+				"stateDescription": {
+					"minimum": 0,
+					"maximum": 150,
+					"step": 1,
+					"pattern": "%.0f %unit%",
+					"readOnly": true
+				},
+				"refresh": true,
+				"actions": []
+			},
+			{
+				"property": "work-mode",
+				"siid": 4,
+				"piid": 1,
+				"friendlyName": "Vacuum Extend - Work Mode",
+				"channel": "work_mode",
+				"type": "Number",
+				"stateDescription": {
+					"minimum": 0,
+					"maximum": 50,
+					"step": 1,
+					"pattern": "%.0f",
+					"readOnly": true
+				},
+				"refresh": true,
+				"actions": []
+			},
+			{
+				"property": "cleaning-time",
+				"siid": 4,
+				"piid": 2,
+				"friendlyName": "Vacuum Extend - Cleaning Time",
+				"channel": "cleaning_time",
+				"type": "Number:Time",
+				"unit": "minutes",
+				"stateDescription": {
+					"minimum": 0,
+					"maximum": 32767,
+					"step": 1,
+					"pattern": "%.0f %unit%",
+					"readOnly": true
+				},
+				"refresh": true,
+				"actions": []
+			},
+			{
+				"property": "cleaning-area",
+				"siid": 4,
+				"piid": 3,
+				"friendlyName": "Vacuum Extend - Cleaning Area",
+				"channel": "cleaning_area",
+				"type": "Number",
+				"stateDescription": {
+					"minimum": 0,
+					"maximum": 32767,
+					"step": 1,
+					"pattern": "%.0f",
+					"readOnly": true
+				},
+				"refresh": true,
+				"actions": []
+			},
+			{
+				"property": "cleaning-mode",
+				"siid": 4,
+				"piid": 4,
+				"friendlyName": "Vacuum Extend - Cleaning Mode",
+				"channel": "cleaning_mode",
+				"type": "Number",
+				"stateDescription": {
+					"options": [
+						{
+							"value": "0",
+							"label": ""
```

Comment:
Would be good to figure out what is the actual meaning of the value. (e.g. look into the mihome app to see if you can relate this to something you see)
If that is not clear please make the label match the value so that at least it is not a bunch of empty entries that you see.
Note this applies also to option 1,2,3